plugin/SDKWrapperAddCurlOption.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+/**
+ *
+ *    Copyright (C) 2016 onOffice Software AG
+ *
+ *    This program is free software: you can redistribute it and/or modify
+ *    it under the terms of the GNU Affero General Public License as published by
+ *    the Free Software Foundation, either version 3 of the License, or
+ *    (at your option) any later version.
+ *
+ *    This program is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *    GNU Affero General Public License for more details.
+ *
+ *    You should have received a copy of the GNU Affero General Public License
+ *    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+/**
+ *
+ * @url http://www.onoffice.de
+ * @copyright 2003-2015, onOffice(R) Software AG
+ *
+ */
+
+namespace onOffice\WPlugin;
+
+use onOffice\SDK\onOfficeSDK;
+use onOffice\WPlugin\WP\WPOptionWrapperBase;
+use onOffice\WPlugin\WP\WPOptionWrapperDefault;
+
+
+/**
+ *
+ */
+
+class SDKWrapperAddCurlOption extends SDKWrapper
```

Comment:
What exactly is this class used for?

Reply:
@jayay this is an extension with specific parameter on CURL options to verify the latency of the API